package.json

```diff
@@ -24,7 +24,7 @@
   "author": "GitHub",
```

Comment:
please bump the "version" field here on line 2 as appropriate in this file and package-lock.json